fdbserver/VersionedBTree.actor.cpp

```diff
@@ -1927,7 +1937,7 @@ class ObjectCache : NonCopyable {
 
 		evictionOrder.clear();
 		cache.clear();
-
+		self->currentSize = 0; // TODO: check if the currentSize should be 0?
```

Comment:
Eviction should reduce currentSize by the evicted item size, so yes `ASSERT` that currentSize is 0 rather than just setting it so we can catch any bugs around this, which would be BTree bugs because no Superpage should ever change size while the front() page ID is the same.